src/ContainerRegistry/ContainerRegistry.Autorest/custom/Get-AzContainerRegistryWebhookEvent.ps1

```diff
@@ -23,13 +23,12 @@ Lists recent events for the specified webhook.
 Get-AzContainerRegistryWebhookEvent  -ResourceGroupName lnxtest -RegistryName lnxcr -WebhookName webhook001
 
 .Outputs
-Microsoft.Azure.PowerShell.Cmdlets.ContainerRegistry.Models.Api202301Preview.IEvent
+Microsoft.Azure.PowerShell.Cmdlets.ContainerRegistry.Models.IEvent
 .Link
 https://learn.microsoft.com/powershell/module/az.containerregistry/get-azcontainerregistrywebhookevent
 #>
 function Get-AzContainerRegistryWebhookEvent {
-[OutputType([Microsoft.Azure.PowerShell.Cmdlets.ContainerRegistry.Models.Api202301Preview.IEvent])]
-[Microsoft.Azure.PowerShell.Cmdlets.ContainerRegistry.Runtime.OutputBreakingChangeAttribute("Microsoft.Azure.PowerShell.Cmdlets.ContainerRegistry.Models.Api202301Preview.IEvent", "15.0.0", "9.0.0", "2025/11/03", ReplacementCmdletOutputType = "Microsoft.Azure.PowerShell.Cmdlets.ContainerRegistry.Models.IEvent", DeprecatedOutputProperties = ("ContentTimestamp, TargetSize, TargetLength, TargetTag, TargetRepository, TargetName, TargetMediaType, TargetDigest, SourceInstanceId, SourceAddr, ActorName, RequestUseragent, RequestMethod, RequestId, RequestHost, RequestAddr, ContentId, ContentAction, TargetUrl, TargetVersion, Action"), NewOutputProperties = ("ContentTimestamp, TargetSize, TargetLength, TargetTag, TargetRepository, TargetName, TargetMediaType, TargetDigest, SourceInstanceId, SourceAddr, ActorName, RequestUseragent, RequestMethod, RequestId, RequestHost, RequestAddr, ContentId, ContentAction, TargetUrl, TargetVersion, Action. This parameter will be changed from single object to 'List'."))]
```

Comment:
This breaking change is not expected, which is caused by removing `inlining-threshold: 100`